File: from-0101-to-0200/0150-evaluate-reverse-polish-notation/solution.go
func evalRPN(tokens []string) int {
    stack := []int{}
    var op1, op2 int

    for _, token := range tokens {
        if token == "+" || token == "-" || token == "*" || token == "/" {
            op1 = stack[len(stack) - 1]
            op2 = stack[len(stack) - 2]
            stack = stack[:len(stack) - 2]
            if token == "+" {
                op1 = op1 + op2
            } else if token == "-" {
                op1 = op2 - op1
            } else if token == "*" {
                op1 = op1 * op2
            } else if token == "/" {
                op1 = op2 / op1
            } 
        } else {
            op1, _ = strconv.Atoi(token)
        }
        stack = append(stack, op1)
    }
    return stack[0]
}

File: from-0101-to-0200/0150-evaluate-reverse-polish-notation/solution.py
class Solution:
    def evalRPN(self, tokens: List[str]) -> int:
        stack = []
        for token in tokens: 
            if token in "+-*/":            
                op1 = stack.pop()
                op2 = stack.pop()
                
                if token == '+':
                    op1 = op1 + op2
                elif token == '-':
                    op1 = op2 - op1
                elif token == '*':
                    op1 = op1 * op2
                else:
                    op1 = int(op2/op1)
                    
                stack.append(op1)
                
            else:
                stack.append(int(token))
                
        return stack.pop()